ggml-vulkan.cpp

```diff
@@ -0,0 +1,3095 @@
+#include "ggml-vulkan.h"
+
+#ifdef VK_CHK_KERNEL
+#include <cblas.h>
+#include <chrono>
+#endif
+
+#ifdef VK_PROFILE
+#define PROFILE(name, block) do { \
+    auto begin = std::chrono::high_resolution_clock::now(); \
```

Comment:
never use `high_resolution_clock`, always use `steady_clock` instead

Reply:
This code isn't used anymore, I may just remove it. But thanks for the hint.